app/controllers/admin/api/access_tokens_controller.rb

```diff
@@ -12,16 +12,33 @@ class Admin::Api::AccessTokensController < Admin::Api::BaseController
 
   before_action :authorize_access_tokens
 
+  ##~ sapi = source2swagger.namespace("Account Management API")
+  ##~ e = sapi.apis.add
+  ##~ e.path = "/admin/api/users/{user_id}/access_tokens.json"
+  ##~ e.responseClass = "List[access_token]"
```

Comment:
I think it should  be only access_token

Reply:
![image](https://user-images.githubusercontent.com/11318903/50590738-70ab5e00-0e8c-11e9-881d-3215f734a8ca.png)

https://github.com/3scale/zync/blob/90c81e21ec2024e5de9acd4b5035c2f6682376f3/app/models/proxy.rb#L1-L6